ur_description/urdf/ur10.urdf.xacro

```diff
@@ -308,8 +308,8 @@
                  degrees)
       -->
       <origin xyz="0 0 0" rpy="0 0 ${-pi}"/>
-      <parent link="${prefix}base_link"/>
-      <child link="${prefix}base"/>
+      <parent link="${prefix}base"/>
+      <child link="${prefix}base_link"/>
```

Comment:
Were these just wrong before? ditto for subsequent similar changes.

Reply:
No, but this is required to have base be the fixed frame and not base_link.

Reply:
This change has been made everywhere, across the board.

Reply:
Thanks for clarifying!